container.go

```diff
@@ -14,37 +14,50 @@ type Container struct {
 	Objects []CanvasObject // The set of CanvasObjects this container holds
 }
 
-func (c *Container) layout() {
-	if c.Layout != nil {
-		c.Layout.Layout(c.Objects, c.size)
-		return
-	}
+// NewContainer returns a new Container instance holding the specified CanvasObjects.
+//
+// Deprecated: Use NewContainerWithoutLayout to create a container that uses manual layout.
+func NewContainer(objects ...CanvasObject) *Container {
+	return NewContainerWithoutLayout(objects...)
 }
 
-// Size returns the current size of this container.
-func (c *Container) Size() Size {
-	return c.size
+// NewContainerWithoutLayout returns a new Container instance holding the specified CanvasObjects
+// that are manually arranged.
+func NewContainerWithoutLayout(objects ...CanvasObject) *Container {
+	ret := &Container{
+		Objects: objects,
+	}
+
+	ret.size = ret.MinSize()
+	ret.layout()
```

Comment:
Like stated in Slack, can probably be removed ;)

Reply:
Fixed, thanks